### PR TITLE
Create /appliances/vm page

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Create /appliances/virtual-machines page
- Added a few strips including one for the /appliance page when we get there
- renamed _docs/ as '_' files don't render anymore to 0_docs/
- Hide /appliance/docs and /appliance/tutorials for now

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/appliances/virtual-machines
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [design](https://app.zeplin.io/project/5eb3d16a91c4117789940141/screen/5ee24a3e2150d5b8fdea8086) and the [copy doc](https://docs.google.com/document/d/1AdLZ6jhGhKaYg6TSnR_Xl8ivRrburrsA8161t_Pk2q8/edit#)

### CODE REVIEW

**Comment by anthonydillon** [comment]
@pmahnke I think the link is /appliance/vm?

**Comment by pmahnke** [comment]
@anthonydillon the doc had virtual-machines, I will check with Rhys

**Comment by pmahnke** [comment]
@anthonydillon renamed

File: navigation.yaml [hunk 1]
```diff
@@ -30,6 +30,10 @@ appliance:
     - title: Docs
       path: /appliance/docs
       persist: True
+      hidden: True
```
Comment (anthonydillon) [inline navigation.yaml:33]: Why hidden? Shouldn't we just remove all docs?

File: navigation.yaml [hunk 2]
```diff
@@ -1158,19 +1162,19 @@ pricing:
 
 documentation:
   title: Docs
-  path: /_docs
+  path: /0_docs
```
Comment (anthonydillon) [inline navigation.yaml:1165]: Why change this url?

Comment (pmahnke) [reply navigation.yaml:1165]: _docs has an underscore and toto and robin stopped underscore docs from being rendered by flask

**Comment by pmahnke** [comment]
@added strips that we need for the homepage as I was at it.

**Pull request merged**